docs/product/components/popovers.html

```diff
@@ -281,6 +287,97 @@
+                             <div class="s-popover--arrow"></div>
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+                        </div>
+                    </div>
+                </div>
+                <p class="fc-light">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla et metus molestie nulla luctus sodales ac luctus justo. Aenean iaculis ac ante sit amet aliquam. Duis dolor velit, imperdiet sed mauris eu, sollicitudin egestas nisl. Ut vitae nulla eu risus iaculis semper sit amet vitae tortor. Sed convallis lacus quis libero placerat finibus. Phasellus pulvinar vel nunc eu tempor. Sed pharetra magna a felis egestas placerat. Sed imperdiet dui a sem fermentum, eget consectetur elit feugiat. Phasellus non condimentum orci. Nam id molestie elit, ut gravida metus. Vivamus vel nunc risus. Maecenas posuere sit amet tellus vel laoreet. Nulla lacus mauris, rhoncus eu laoreet quis, mattis vehicula nisl. Integer efficitur quam et nisi luctus scelerisque. Mauris efficitur lectus ac malesuada congue.</p>
+                <button class="s-btn s-btn__link" data-action="section#showHelp">Click here for help</button>
+            </div>
+        </div>
+    </div>
+    <script>
+        window.addEventListener("load", () => {
+                Stacks.addController("section", {
+                targets: ["help"],
+                showHelp(e) { Stacks.showPopover(this.helpTarget); e.stopPropagation(); },
+                hideHelp(e) { Stacks.hidePopover(this.helpTarget); },
+            });
+        });
+    </script>
+
+    {% header "h4", "JavaScript configuration" %}
+    <p class="stacks-copy">Situations may also arise where popovers need to be attached to an element after the document is rendered. For example, a button could have a contextual menu that is too expensive to serve up on every page load.</p>
+    <p class="stacks-copy">Popovers can be attached to an element after the fact using <code class="stacks-code">Stacks.attachPopover</code>.</p>
+    <p class="stacks-copy">This method takes three parameters, the element to attach the popover to, the popover either as an element or an HTML string, and optional options for displaying the popover.</p>
+
+    <div class="stacks-preview">
+{% highlight javascript %}
+Stacks.application.register("actions", class extends Stacks.StacksController {
+
+    var loaded = false;
+
+    async load() {
+        if (this.loaded) { return; }
+        Stacks.attachPopover(this.element,
+                             await fetch(`/posts/{postId}/actions`),
+                             { autoShow: true, toggleOnClick: true });
+        this.loaded = true;
+    }
+});
+{% endhighlight %}
+    <div class="stacks-preview--example bg-black-025">
+        <button class="s-btn s-btn__outlined" data-controller="actions" data-action="actions#load">Show actions for this post</button>
+    </div>
+</div>
+<script>
+    window.addEventListener("load", () => {
```

Comment:
Same.  Should these move into a .js file?

Reply:
Added to `feature.popovers.js`